src/bitcoinrpc.cpp

```diff
@@ -448,7 +448,6 @@ static const CRPCCommand vRPCCommands[] =
     { "verifymessage",          &verifymessage,          false,     false },
     { "getwork",                &getwork,                true,      false },
     { "listaccounts",           &listaccounts,           false,     false },
-    { "settxfee",               &settxfee,               false,     false },
```

Comment:
there's really no need to remove that rpc call, just remove it from gui part if you must.